src/shared/Core/EncodingEx.cs

```diff
@@ -0,0 +1,8 @@
+using System.Text;
+
+namespace GitCredentialManager;
+
+public static class EncodingEx
```

Comment:
Having a little trouble parsing the `Ex` piece of this name, although I'm sure there's a good reason why it was chosen.

Reply:
`System.Encoding` is a class that already exists. It's an old Windows convention to 'extend' a sealed/static class or function by giving it an 'Ex' suffix to signify some extra functionality.

Examples already in GCM: `ConsoleEx`, `BoolConvertersEx` and `AssertEx`.